modules/representation_management/spec/models/representation_management/form_2122_base_spec.rb

```diff
@@ -71,6 +71,48 @@
     it { expect(subject_with_claimant).not_to allow_value('123456789A').for(:claimant_phone) }
     it { expect(subject_with_claimant).not_to allow_value('123456789').for(:claimant_phone) }
 
+    describe 'representative_phone' do
+      context 'when representative is an instance of AccreditedIndividual' do
+        it 'returns #phone of the representative' do
+          representative = create(:accredited_individual, phone: '5555555555')
+          subject.representative_id = representative.id
+          expect(subject.representative_phone).to eq(representative.phone)
+        end
+      end
+
+      context 'when representative is an instance of Veteran::Service::Representative' do
+        it 'returns #phone_number of the representative' do
+          representative = create(:representative, phone_number: '5555555555')
+          subject.representative_id = representative.representative_id
+          expect(subject.representative_phone).to eq(representative.phone_number)
+        end
+      end
+    end
+
+    describe 'representative_individual_type' do
+      context 'when representative is an instance of AccreditedIndividual' do
+        it 'returns #individual_type of the representative' do
+          representative = create(:accredited_individual, individual_type: 'attorney')
+          subject.representative_id = representative.id
+          expect(subject.representative_individual_type).to eq(representative.individual_type)
+        end
+      end
+
+      context 'when representative is an instance of Veteran::Service::Representative' do
+        it 'returns the first element in the user_types array' do
+          representative = create(:representative, user_types: %w[attorney claims_agent])
```

Comment:
`claims_agent` should be `claim_agents` for this factory

Reply:
Done.